### PR TITLE
fix issue #27

### PR DESCRIPTION
**Pull request by giabao**
pls see commit messages

### CODE REVIEW

**Comment by pivotal-issuemaster** [comment]
@giabao Please sign the [Contributor License Agreement](https://cla.pivotal.io/sign/reactor?repositoryId=reactor/reactor-scala-extensions&pullRequestId=44)!

[Click here](https://cla.pivotal.io/sync/reactor?repositoryId=reactor/reactor-scala-extensions&pullRequestId=44) to manually synchronize the status of this Pull Request.

See the [FAQ](https://cla.pivotal.io/about) for frequently asked questions.

File: build.gradle [hunk 1]
```diff
@@ -15,13 +15,18 @@ apply plugin: 'maven-publish'
 repositories {
     mavenCentral()
 }
-
+configurations {
+    scalaAnt
+}
 dependencies {
+    scalaAnt 'com.sandinh:scala-ant_2.13:2.13.1'
```
Comment (giabao) [inline build.gradle]: [scala-ant repository](https://github.com/sandinh/scala-ant)

**Comment by pivotal-issuemaster** [comment]
@giabao Thank you for signing the [Contributor License Agreement](https://cla.pivotal.io/sign/reactor?repositoryId=reactor/reactor-scala-extensions&pullRequestId=44)!

**Comment by giabao** [comment]
@sinwe OK. I will make all test green and create a PR to branch sreator_2.13

Note also issue #46

**Comment by sinwe** [comment]
@giabao I've cherry-picked this PR commits to my original sreactor_213 and managed to make it passed.
I'll merge this PR and force merge my branch that contains your commits. That's really helpful.
In the meantime, appreciate if you can fix #46

**Pull request merged**